packages/core/src/uma.ts

```diff
@@ -358,6 +352,11 @@ export function isValidUmaAddress(umaAddress: string) {
     return false;
   }
 
+  const domain = addressParts[1];
+  if (!domain.match(/^[a-z0-9-\.]+$/i)) {
```

Comment:
I think this will match even a string without  a `.` . are uma domains this flexible?

Reply:
By adding this I was matching closer to the requirements in one place on the sparkcore side https://github.com/lightsparkdev/webdev/blob/main/sparkcore/sparkcore/ent/schema/umame/umame_utils.py and preventing eg `$bob@`

But yeah I guess `$bob@localhost` seems like it should be valid?